### ISSUE TITLE
mxnet website should link the branding guidelines from apache incubator

### ISSUE THREAD

**Issue by szha** [apache/incubator-mxnet#15162]
related guidelines:
- http://apache.org/foundation/press/Project-PR-BrandGuidelines.pdf
- Linked to from http://apache.org/press/
- https://apache.org/foundation/marks/
- https://www.confluent.io/apache-guidelines

**Comment by ys2843** [comment]
Is this issue still valid? Can you give more info about this requirement? thanks

**Comment by szha** [comment]
Yes, this is still a valid issue. We need to create a page in the community on the branding requirements when depending on our project
